frontend/src/components/program/DialogActivityCreate.vue

```diff
@@ -191,7 +191,7 @@ export default {
       if (showDialog) {
         this.refreshCopyActivitySource()
         this.setEntityData({
-          title: this.entityData?.title || this.$tc('entity.activity.new'),
+          title: this.entityData?.title,
```

Comment:
If you remove this text, then I think you should set it as the dialog title. Otherwise it is not entirely clear what you are working on.

Reply:
Sorry I must have edited a existing activity. There it is missing.